public/app/plugins/panel/candlestick/CandlestickPanel.tsx

```diff
@@ -260,19 +282,11 @@ export const CandlestickPanel = ({
       replaceVariables={replaceVariables}
       dataLinkPostProcessor={dataLinkPostProcessor}
     >
-      {(uplotConfig, alignedDataFrame) => {
-        alignedDataFrame.fields.forEach((field) => {
-          field.getLinks = getLinksSupplier(
-            alignedDataFrame,
-            field,
-            field.state!.scopedVars!,
-            replaceVariables,
-            timeZone
-          );
-        });
```

Comment:
unrelated cleanup. (forgot to remove in https://github.com/grafana/grafana/pull/83654)